rpm/0053-sailfishos-gecko-Make-fullscreen-enabling-work-as-us.patch

```diff
@@ -1,8 +1,8 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Raine Makelainen <raine.makelainen@jolla.com>
 Date: Wed, 19 Aug 2020 14:27:21 +0300
-Subject: [PATCH] [sailfishos][gecko] Make fullscreen enabling work as used to
- with pref full-screen-api.content-only. Fixes JB#44129
+Subject: [PATCH 1/1] [sailfishos][gecko] Make fullscreen enabling work as used
```

Comment:
git format-patch --zero-commit -N sha1

Reply:
Thanks again; I've changed this one too.